src/adapters/adapter.ts

```diff
@@ -71,5 +72,22 @@ export abstract class Adapter {
    */
   public addModel(model: typeof Model): void {
     model.adapter = this;
+    this.models.push(model);
+  }
+
+  /**
+   * Returns an array containing all classes of the Models registered with 'addModel'.
+   */
+  public getAllModels(): Array<typeof Model> {
+    return this.models;
+  }
+  
+  /**
+   * Truncates all registered model tables with 'Model.truncate'.
+   */
+  public truncateAllModels() {
```

Comment:
This method needs to return a promise, so that we can wait until the tables are truncated before we do something else.

Example:

```ts
public async truncateAllModels(): Promise<void> {
  await Promise.all(this.models.map((model) => model.truncate()));
}
```

Reply:
You're right, I'll fix here.